lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
Or add another validator so we've got one for optional-file validation and one for required-file validation?